ring/ring.go

```diff
@@ -101,24 +102,52 @@ func (r *Ring) loop() {
 	})
 }
 
-// Get returns a collector close to the hash in the circle.
-func (r *Ring) Get(key uint32) (IngesterDesc, error) {
+// Get returns up to n ingesters close to the hash in the circle.
```

Comment:
I wonder if it's worth having a new type that implements Collector but has multiple ingesters as a backend?